tests/dfomp2-4/input.dat

```diff
@@ -18,10 +18,11 @@ set {
   guess sad
   reference uhf
   mp2_type df
+  rms_mograd_convergence 5
```

Comment:
can do without this addition now?

Reply:
Oh, _this_ test. Message me on Slack when you want to talk about this one. GitHub isn't good for the back-and-forth that discussing this one will require.

Reply:
For future reference: this test is pathological, so we can't be sure if we have the energy to six decimal places or not. Lori and I agreed to lower the convergence check to 5 decimals.